### ISSUE TITLE
Create ReviewHome (container) with not expandable reviews - should list each review for the User

### ISSUE THREAD

**Issue by nmadruga** [openmsupply/application-manager-web-app#379]

**Comment by andreievg** [comment]
@nmadruga do you mind if someone takes this issue over while you are away ?

**Comment by nmadruga** [comment]
No, all good by me. I was keen on working on it but really shouldn't so I'm happy to work on what's available when I'm ready to do more coding - probably next week.

For now I'll focus in reviews and meetings ☺️

**Comment by andreievg** [comment]
There is a slight refinement (maybe not so slight).
https://app.diagrams.net/#G1-kKRiGtnZ8__cL82EOdpYap7Rv79GBvX

But the idea is it makes a lot more sense to display concurrent tasks for the same section group by section.

I am going to have a go at this one now
